### ISSUE TITLE
publish Turtle

### ISSUE THREAD

**Issue by VladimirAlexiev** [OP-TED/ePO#392]
@costezki , @giorgialodi Please publish Turtle for all rdf files in https://github.com/OP-TED/ePO/tree/master/implementation.
Turtle is much easier to read and discuss than RDF/XML.
Also, it makes it easier to spot problems.

Easy to do with eg
```
riot --formatted=turtle ePO_owl_core.rdf  1>ePO_owl_core.ttl
riot --formatted=turtle ePO_restrictions.rdf  1>ePO_restrictions.ttl
```

**Comment by giorgialodi** [comment]
I totally agree with you @VladimirAlexiev about Turtle. I typically convert RDF/XML in Turtle, but here I do not decide :) I think that @muricna and @costezki can definitely take into account your comment :)

**Comment by VladimirAlexiev** [comment]
@muricna and @costezki please note the following issues that I was able to easily find in the Turtle rendition

**Comment by andreea-pasare** [comment]
This feature is already implemented in model2owl and will be available in ePO release 3.1.0.
